apollo-server/v4/integration-testing/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "as4-integration-testing",
+  "version": "1.0.0",
+  "description": "An example of integration testing with Apollo Server 4",
+  "main": "dist/index.js",
+  "type": "module",
+  "license": "MIT",
+  "scripts": {
+    "test": "npm run compile && jest",
+    "postinstall": "npm run compile",
+    "compile": "tsc",
+    "start": "npm run compile && node ./dist/server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "dependencies": {
+    "@apollo/server": "^4.0.0-alpha.1",
```

Comment:
you could make this `latest` for now, since we are publishing the alphas to that tag